scan-framework/src/main/java/com/getbouncer/scan/framework/Loader.kt

```diff
@@ -99,14 +100,27 @@ abstract class WebLoader(private val context: Context) : Loader {
             return null
         }
 
-        try {
-            retry(NetworkConfig.retryDelay) { downloadAndVerify() }
+        val exception = try {
+            retry(NetworkConfig.retryDelay) {
```

Comment:
Instead of this logic, would it make more sense to add a `doNotRetry` or `onlyRetry` list of exception classes to the `retry` method signature?

is this something you think we'll use in other cases?

Reply:
yeah, thought about it. could be useful making it more generic, but i'm going to save that for another day as we're trying to hit a deadline